File: src/program_structures/decision/c/if_else_if_statement/sample_if_else_if_statement.c
/*
 * Author: Jhonatan Casale (jhc)
 *
 * Contact : jhonatan@jhonatancasale.com
 *         : casale.jhon@gmail.com
 *         : https://github.com/jhonatancasale
 *         : https://twitter.com/jhonatancasale
 *         : http://jhonatancasale.github.io/
 *
 * Create date Wed 23 Nov 03:25:28 BRST 2016
 *
 */

#include <stdlib.h>
#include <stdio.h>

int main (int argc, char **argv)
{
  int i = -1;

  if ( i > 0 ) { 
    printf ( "> 0\n" ); 
  } else if ( i == 0 ) { 
    printf ( "== 0\n" ); 
  } else { 
    printf ( "< 0\n" ); 
  }

  return (EXIT_SUCCESS);
}

File: src/program_structures/decision/r/if_else_if_statement/sample_if_else_if_statement.R
#
#  Author: Jhonatan Casale (jhc)
# 
#  Contact : jhonatan@jhonatancasale.com
#          : casale.jhon@gmail.com
#          : https:#github.com/jhonatancasale
#          : https:#twitter.com/jhonatancasale
#          : http:#jhonatancasale.github.io/
# 
#  Create date Sat 26 Nov 04:27:37 BRST 2016
#

i <- -1

if (i > 0) { 
  print ('> 0\n') 
} else if (i == 0) { 
  print ('== 0\n') 
} else { 
  print ('< 0\n') 
}

# for single line statements the {} is nearly optional
if (i > 0) {
  print ('> 0\n') 
} else if (i == 0) {
  print ('== 0\n') 
} else print ('< 0\n') 

# for really single line statements the {} is nearly optional
if (i > 0) print ('> 0\n') else if (i == 0) print ('== 0\n') else print ('< 0\n') 
